amp-camp/src/server/ApiManager.js

```diff
@@ -1,39 +1,66 @@
 class ApiManager {
 
     constructor() {
+        this._validParams = {
```

Comment:
I'm a bit confused about the convention used for properties and function names. I assume starting with an underscore "_" would mean that the property is private, is that correct?

If so, is there any reason why variables like "apiCategoryEndpoint" don't follow the same convention?
Also, should this apply to private functions as well?

Reply:
Good point. I started doing the underscore thing, but I didn't do it consistently.

I'll change that!